checks/pinned_dependencies.go

```diff
@@ -602,6 +602,14 @@ func validateGitHubActionWorkflow(pathfn string, content []byte,
 				// Cannot check further, continue.
 				continue
 			}
+
+			// nolint:lll
+			// Check whether this is an action defined in the same repo,
+			// https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#referencing-an-action-in-the-same-repository-where-a-workflow-file-uses-the-action.
+			if strings.HasPrefix(execAction.Uses.Value, "./") {
```

Comment:
Thanks! As far as I understand it's possible to reference local actions using "{owner}/{repo}" but I don't think I have ever seen anyone really use it (probably because "./" is shorter :-)

Reply:
good point. Hopefully this will encourage users to use the right idiom:-)